doc/GoogleSheetsConnector.md

```diff
@@ -134,8 +135,21 @@ myGoogleSheetsConnector.on({sheetIdOrTitle: 0}, myHandler)
 
 /** Check for changes only in a specific sheet by title and every 30 seconds */
 myGoogleSheetsConnector.on({sheetIdOrTitle: 'Sheet1', interval: 30 * 1000}, myHandler)
+
+/** Check for changes using a key column for more accuracy, see Known Issues below */
+myGoogleSheetsConnector.on({keyColumn: 'Identifier'}, myHandler)
 ```
 
+##### Known Issues:
+Since Google Sheets does not provide rows identifiers, it is hard to identify which events caused the rows changes,for example when deleting a row which is not at the bottom of the spreadsheet, the rows below are shifted up. This can be interpritated as an update of all the rows below including the "deleted" row and a deletion of the last row.
+These are the options to handle such cases:
+1. Add or delete rows just from the bottom of the spreadsheet.
+2. Consider the `event.oldRows` & `event.newRows`, these attributes will provide you all the information about the spreadsheet rows before and after the changes. Now it is up to you to decide how to handle the events.
```

Comment:
Treat `event.oldRows` and `event.newRows` as before and after snapshots. You may compare these yourself to ascertain which changes your code is interested in.